packages/ui/packages/InputRange/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+name: InputRange
+route: /ui/input-range
+menu: 3.Components
+---
+import { Playground, Props } from 'docz';
+import InputRange from './src/index';
+
+# Basic Usage
+
+<Playground>
+<InputRange
+    className='foobar'
+    fromTitle="from"
+    toTitle="to"
+    onChangeComplete={value => console.log(value)}
```

Comment:
fix indent, there is a plugin which written for `vscode` but also works on `phpstorm`, could use that one to indent.